sycl/test-e2e/no-xfail-without-tracker.cpp

```diff
@@ -19,15 +19,19 @@
 // - in the result, search for "XFAIL" again, but invert the result - this
 //   allows us to get the line *after* XFAIL
 // - in those lines, check that XFAIL-TRACKER is present and correct. Once
-//   again, invert the search to get all "bad" lines
+//   again, invert the search to get all "bad" lines and save the test names in
+//   improperly-xfailed-tests.txt
 // - make a final count of how many ill-formatted directives there are and
 //   verify that against the reference
+// - ...and check if the list of improperly "xfailed" tests needs to be updated.
 //
-// RUN: grep -rI "XFAIL:" %S -A 1 --include=*.c --include=*.cpp \
-// RUN:     --no-group-separator | \
+// RUN: grep -rI "XFAIL:" %S -A 1 --exclude=%s --include=*.c --include=*.cpp \
+// RUN:     --exclude=no-xfail-without-tracker.cpp --no-group-separator | \
```

Comment:
Why do we need both `%s` and the name? `%s` should be an absolute path to the current file anyway

Reply:
Just forgot to delete - f0ea4c72b6e5056b85cbb84d914fd033e3d6168a